### ISSUE TITLE
bootstrap theme input

### ISSUE THREAD

**Issue by amazingSurge** [alertifyjs/alertify.js#52]
When use bootstrap theme, please updaste input as well.

```
case "bootstrap":
    this.dialogs.input = "<input class="form-control" type='text'>";
```

**Comment by bradberger** [comment]
Definitely, have added it to the list for the next release

**Comment by bradberger** [comment]
@amazingSurge This just landed in `v1.0.5`

**Comment by amazingSurge** [comment]
Good work. Thank you!
